benchmark/indexselect.cpp

```diff
@@ -292,29 +270,10 @@ static void NvFuserScheduler_IndexSelect(
   at::Tensor t2 =
       (select_dim ? at::randn({nFeat, select_size}, options)
                   : at::randn({select_size, nFeat}, options));
-  fusion_executor_cache->profile(true);
-  fusion_executor_cache->runFusionWithInputs({t2, t0, t1});
-
-  auto compile_log = fusion_executor_cache->getMostRecentExecutorInfo();
-  auto executor_instance = compile_log.fusion_executor;
-  auto params = toString(compile_log.params);
-  auto lparams = toString(compile_log.fusion_executor->lastLaunchParams());
 
-  benchmark_state.SetLabel(params + lparams);
+  std::vector<c10::IValue> aten_inputs = {t2, t0, t1};
```

Comment:
why `{t2, t0, t1};` instad of `{t0, t1, t2};`

Reply:
I don't know. I thought I didn't change how the benchmark was originally done.